docs/api.md

```diff
@@ -67,7 +67,17 @@ Use `prettier.resolveConfig.sync(filePath [, options])` if you'd like to use syn
 
 The promise will be rejected if there was an error parsing the configuration file.
 
-If `options.useCache` is `false`, all caching will be bypassed.
+If `filePath` is provided, the path of the configuration file in the project where `filePath` exists will be returned, otherwise the path of the configuration file in the current project will be returned.
```

Comment:
I think it would be better to link to the cosmiconfig docs for details on how the resolving works: https://github.com/davidtheclark/cosmiconfig#explorersearch

Reply:
Fixed:+1: